solvebio/resource/dataset.py

```diff
@@ -22,75 +22,61 @@ class Dataset(CreateableAPIResource,
               UpdateableAPIResource):
     """
     Datasets are access points to data. Dataset names are unique
-    within versions of a depository.
+    within a vault folder.
     """
+    USES_V2_ENDPOINT = True
```

Comment:
can we rename this to: `RESOURCE_VERSION = 2` or something? this way we can update these as we add more versions without adding new flags